__tests__/endpoints/app.test.js

```diff
@@ -61,3 +61,47 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("status 200, return the object article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      })
+      .catch((err) => {
+        req.status(err.status).send({ msg: err.msg });
+      });
+  });
+
+  test("status 400, returns Bad Request when the article_id passed it is a different data type", () => {
+    return request(app)
+      .get("/api/articles/cat")
+      .expect(400)
+      .then(({ error }) => {
+        expect(JSON.parse(error.text).msg).toBe("Bad Request");
```

Comment:
You shouldn't need to parse the response here. If you just do `expect(error.msg).toBe` this should work 👍